### PR TITLE
rework formatting with cmake-format

### LINKED ISSUE #4621: cmakify mangles comments
our cmake files have comments that depend on their current formatting for clarity, the cmake-format script instead merges their formatting to become a single line, we should either make the comments not sensitive to autoformatting or change the settings of cmake-format.

It's important we do this before other changes to the cmake files as we will have to revert the cmake files to a previous version in order to restore their formatting.

### PR DESCRIPTION
**Pull request by ebbit1q**
## Related Ticket(s)
- Fixes #4621

## Short roundup of the initial problem
the cmake-format behaved weird

## What will change with this Pull Request?
- clangify.sh is now format.sh 
- format.sh --cmake additionally performs cmake-format on all edited cmake files
- cmakify.sh is has been removed
- the ci will format cmake files
- the ci has been updated to 22.04 beta in order to easily install cmake-format
- comments in cmake files are no longer formatted
- dbconverter has been added to the locations to format
- all files have been formatted with the new script, including some that have been missed previously

### CODE REVIEW

File: .ci/lint_cpp.sh [hunk 1]
```diff
@@ -10,10 +10,10 @@ if ! git merge-base origin/master HEAD; then
   git fetch --unshallow
 fi
 
-# Check formatting using clangify
+# Check formatting using format.sh
 echo "Checking your code using clang-format..."
```
Comment (tooomm) [inline .ci/lint_cpp.sh:14]:
```suggestion
echo ""
echo "Checking your code using clang-format and cmake-format..."
```

File: .github/workflows/desktop-lint.yml [hunk 2]
```diff
@@ -9,20 +9,20 @@ on:
...
-    - name: Install clang-format
+    - name: Install dependencies
       shell: bash
       run: |
         sudo apt-get update
-        sudo apt-get install -y --no-install-recommends clang-format
+        sudo apt-get install -y --no-install-recommends clang-format cmake-format
 
-    - name: Run clangify
+    - name: Format code
```
Comment (tooomm) [inline .github/workflows/desktop-lint.yml:26]:
```suggestion
    - name: Check code styles
```

File: .github/workflows/desktop-lint.yml [hunk 3]
```diff
@@ -9,20 +9,20 @@ on:
 
 jobs:
   clang-format:
```
Comment (tooomm) [inline .github/workflows/desktop-lint.yml:11]:
```suggestion
  Check Formatting:
```

**Review by tooomm — COMMENTED** [review]
Left some rewording suggestions.

Most comments use `# comment` style, few are changed to `#comment`.

New contributors might not be aware that they need to pass `--cmake` to the format script, when only seeing
`Please correct it or run the "format.sh" script`. The CONTRIBUTING file does also not mention this.

**Comment by ebbit1q** [comment]
that's because we don't expect new contributors to edit the cmake files, it's fine as is, no need to make things complicated and the script has its own help function

**Comment by tooomm** [comment]
Wouldn't it be useful to print the used version(s) for all runs?
It looks like those are only displayed on runs where a format diff is found.

**Comment by ebbit1q** [comment]
which is fine

**Review by ZeldaZach — APPROVED** [review]

**Pull request merged**